### PR TITLE
Adds grip edge/vertex generation functionality from FHIR files.

### PR DESCRIPTION
**Pull request by matthewpeterkort**
vertex generation test: python -m pytest tests/integration/simplified/test_simplify_task_grip.py -s

`iceberg data simplify tests/fixtures/simplify/synthea grip_data  --schema_path https://raw.githubusercontent.com/bmeg/iceberg/feature/updated-schemas/schemas/graph/graph-fhir.json --dialect GRIP --project_id synthea-test`
works

See https://github.com/bmeg/iceberg-schema-tools/issues/25 for description of use case. (BW)

### CODE REVIEW

File: iceberg_tools/data/simplifier/__init__.py [hunk 1]
```diff
@@ -490,6 +493,35 @@ def validate_simplified_value(v) -> bool:
...
 
 
+def _grip_simplifier(simplified: dict):
+    new_dict = {"gid": simplified["id"], "label": simplified["resourceType"]}
+    del simplified["resourceType"]
+
+    # uuid regex
+    pattern = r'[a-zA-Z]+/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}'
+    names = []
+    print("SIMPLIFIED ORIG: ", simplified)
```
Comment (bwalsh) [inline iceberg_tools/data/simplifier/__init__.py:503]: We should remove debugging statement.

File: iceberg_tools/data/simplifier/__init__.py [hunk 2]
```diff
@@ -490,6 +493,35 @@ def validate_simplified_value(v) -> bool:
...
+            temp_fields = fields[0]
+            if "_" in temp_fields:
+                temp_fields = temp_fields.split("_")[-1]
+
+            names.append(fields[0])
+
+    for elems in names:
+        simplified.pop(elems)
+
+    print("SIMPLIFIED: ", simplified)
```
Comment (bwalsh) [inline iceberg_tools/data/simplifier/__init__.py:520]: same ...

File: iceberg_tools/schema/fhir_resources/binding_lookup.py [hunk 3]
```diff
@@ -88,10 +88,10 @@ def create_fhir_definitions_lookup(iceberg_path: str = "~/.iceberg"):
     connection = sqlite3.connect(path)
 
     with connection:
-        connection.execute(f'DROP table IF EXISTS element_bindings')
-        connection.execute(f'CREATE TABLE if not exists element_bindings (id PRIMARY KEY, entity Text)')
+        connection.execute('DROP table IF EXISTS element_bindings')
```
Comment (bwalsh) [inline iceberg_tools/schema/fhir_resources/binding_lookup.py:91]: Good catch, thanks

**Review by bwalsh — CHANGES_REQUESTED** [review]
Looks good.  One minor comment below re. debugging statements.
See issue, please write up a use case that describes what this is and when / why to use it.

**Comment by matthewpeterkort** [comment]
This got implemented in https://github.com/bmeg/jsonschemagraph instead

**Pull request closed without merging**